packages/core/test-project/ios/.gitignore

```diff
@@ -16,11 +16,9 @@ build/
 xcuserdata
 *.xccheckout
 *.moved-aside
-DerivedData
```

Comment:
is it necessary to remove `DerivedData` here? I believe these are re-generated on each local build so could create noise in git